pom.xml

```diff
@@ -339,7 +339,7 @@
           <tagNameFormat>pipeline-model-definition-@{project.version}</tagNameFormat>
         </configuration>
       </plugin>
-      <plugin> <!-- TODO can probably be deleted -->
+      <plugin>
```

Comment:
Similar to the other TODO: you want to delete it, do a PR (or do the deletion on this one) or file a JIRA. 

I'm happy to merge this PR without the TODO or we can wait and you can determine that this section can be deleted and change to deleting the section.  Either way the TODO doesn't belong.

Reply:
> file a JIRA

Way too heavyweight for a minor bit of code cleanup, and not discoverable from the code itself. Just never mind.